baseplate/clients/sqlalchemy.py

```diff
@@ -191,6 +191,8 @@ def on_before_execute(
 
         trace_name = "{}.{}".format(context_name, "execute")
         span = server_span.make_child(trace_name)
+        if len(statement) > 100:
+            statement = statement[:100]
         span.set_tag("statement", statement)
```

Comment:
```suggestion
        span.set_tag("statement", statement[:100])
```

It's safe to just truncate without checking. `"foo"[:100] == "foo"`.

Reply:
🔕 on the other hand, that's magical and someone in the future might not know about that behavior and be confused.

You might also want to say `statement = statement[:100] + "..."` when you're truncating so it's clear that some of the query is missing.   In the case where it happens to truncate it, say, right before the WHERE clause, having this could really aid in keeping confusion down.

Reply:
yeah fair point about the ellipsis to make it clear it was truncated (or even `[AND %d MORE BYTES]` or something)